src/components/Navigator/NavigatorCardItem.vue

```diff
@@ -182,36 +182,43 @@ export default {
 $item-height: 32px;
 
 .navigator-card-item {
+  --tree-toggle-padding: var(--card-horizontal-spacing-small);
+  --chevron-width: var(--card-horizontal-spacing);
+  --nesting-spacing: var(--card-horizontal-spacing) + var(--card-horizontal-spacing-small);
```

Comment:
This is technically not a valid addition. It only works, because the var is directly replaced with `a + b` in a `calc()`